client/my-sites/theme/main.jsx

```diff
@@ -922,6 +925,7 @@ class ThemeSheet extends Component {
 
 		return (
 			<Main className={ className }>
+				<AsyncLoad require="calypso/components/global-notices" placeholder={ null } id="notices" />
```

Comment:
I'd suggest moving this below the `Query*`, `DocumentHead`, and `PageViewTracker` components. All of them render nothing but perform some sort of queries or actions under the hood. The first time we render something is where we call `this.renderBar()`, so we might move this line right above it.